unlock-app/src/components/creator/FatalError.tsx

```diff
@@ -1,8 +1,9 @@
-import PropTypes from 'prop-types'
-
 import React from 'react'
 import styled from 'styled-components'
 import { ETHEREUM_NETWORKS_NAMES } from '../../constants'
+/* eslint-disable */
```

Comment:
I think I understand - you need to copy the eslint config from paywall. This will fix this annoying problem